tests/MuchAdo.Sqlite.Tests/SqliteTests.cs

```diff
@@ -69,6 +69,23 @@ public void InsertAndSelectNameValue()
 			.Query<NameValue>().Should().Equal(items);
 	}
 
+	[Test]
+	public void NullParameter()
```

Comment:
I can do it in a follow-up PR, but this would be a good check for all of the providers.